newIDE/app/src/EventsSheet/EventsTree/index.js

```diff
@@ -669,9 +673,11 @@ export default class ThemableEventsTree extends Component<
   };
 
   _onEndDrag = () => {
-    this.setState({ draggedNode: null });
-    this._restoreFoldedNodes();
-    this.forceEventsUpdate();
+    if (this.state.draggedNode) {
```

Comment:
Yes! I call this method in `_onDrop` to update the eventsSheet and compute the "nextPosition" for history. Since this method is also called after each drag action (regardless of whether it was successfully dropped or not), I wanted to avoid calling a second time after the drop

Reply:
Ok I let you choose if this is really a same function called at two places, or if it should be separated in two functions/renamed to emphasis their job.
Maybe add a comment at least to explain why this could be called with a dragged event, which sounds like it should not happen but can :)